src/core/AutoRest/Plugins/AzureValidator.cs

```diff
@@ -71,22 +72,26 @@ private void LogValidationMessage(ValidationMessage validationMessage)
     protected override async Task<bool> ProcessInternal()
     {
         var files = await ListInputs();
-        if (files.Length != 1)
+        foreach (var file in files)
         {
-            return false;
-        }
+            var content = await ReadFile(file);
+            var fs = new MemoryFileSystem();
+            fs.WriteAllText(file, content);
 
-        var content = await ReadFile(files[0]);
-        var fs = new MemoryFileSystem();
-        fs.WriteAllText(files[0], content);
+            var serviceDefinition = SwaggerParser.Load(file, fs);
+            var validator = new RecursiveObjectValidator(PropertyNameResolver.JsonName);
+            
+            var metadata = new ServiceDefinitionMetadata
+            {
+                ServiceDefinitionDocumentType = (ServiceDefinitionDocumentType)Enum.Parse(typeof(ServiceDefinitionDocumentType), (await GetValue("openapi-type"))?.ToString() ?? ServiceDefinitionDocumentType.ARM.ToString(), true),
```

Comment:
1. It can be either from the default config file or the command line. Command line arguments trump whatever is specified in the config file provided we also pass in the correct scope.
2. ```openapi-type``` describes the type of specification provided while ```ServiceDefinitionDocumentType``` is more like metadata for the service document object generated from the provided spec, hence the naming.